### PR TITLE
Initial Deployment Strategy

### PR DESCRIPTION
**Pull request by ifilonenko**
Kubernetes Dev Ops Strategy.

Tested via step-by-step execution elaborated up on in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+cluster: Running
+kubectl: Correctly Configured: pointing to minikube-vm at 192.168.99.100
+```
+You now have a single-noded Kubernetes cluster
+
+## 3. Build Docker image and deploy to your Minikube environment
+```bash
+> minikube docker-env
+export DOCKER_TLS_VERIFY="1"
+export DOCKER_HOST="tcp://192.168.99.100:2376"
```
Comment (kevingreer) [inline README.md]: is this IP significant?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+export DOCKER_CERT_PATH="/Users/ilanfilonenko/.minikube/certs"
```
Comment (kevingreer) [inline README.md]: might want to change to be generic

File: README.md (hunk 1 above)
Comment (ifilonenko) [reply README.md]: This is an example bash command of an IP that docker sits on in your own subnet. It is almost standard to be that one. The IP itself isn't significant.

File: README.md (hunk 2 above)
Comment (ifilonenko) [reply README.md]: ohhh good point

**Pull request merged**